include/bio/version.hpp

```diff
@@ -17,11 +17,11 @@
  */
 
 //!\brief The major version as MACRO.
-#define BIO_VERSION_MAJOR     0
+#define BIO_VERSION_MAJOR 0
```

Comment:
Du und dein align Fimmel :rofl:

Reply:
This is auto-formatted by clang-format now. Are you sure you are not breaking that?